source/appModules/searchui.py

```diff
@@ -20,6 +20,12 @@ class StartMenuSearchField(SearchField):
 	# #7370: do not announce text when start menu (searchui) closes.
 	announceNewLineText = False
 
+	def _get_description(self):

```

Comment:
Is this the correct type?

```suggestion
	def _get_description(self) -> str:
```